.github/workflows/lint.yaml

```diff
@@ -43,3 +43,20 @@ jobs:
         run: python -m pip install gitlint
       - name: Run gitlint check
         run: gitlint --commits origin/${{ github.event.pull_request.base.ref }}..HEAD
+  checkton:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run Checkton
+        id: checkton
+        uses: chmeliik/checkton@main
+        with:
+          fail-on-findings: false
+      - name: Upload SARIF File
```

Comment:
What does uploading the file get us? Any idea?

Reply:
@johnbieren The Checkton only finds the linting problems, so uploading the results to SARIF will create warning messages on the PR.  Example of it working [ PR here ](https://github.com/seanconroy2021/shellchecktest/pull/6)

Reply:
ah, okay cool